rerun_py/rerun_bindings/rerun_bindings.pyi

```diff
@@ -70,7 +71,7 @@ class RRDArchive:
     def num_recordings(self) -> int: ...
     def all_recordings(self) -> list[Recording]: ...
 
-def load_recording(filename: str) -> Recording:
+def load_recording(filename: str | os.PathLike) -> Recording:
```

Comment:
I had a doubt whether `str` was implied by `PathLike` but it indeed does not. 👍🏻